src/components/HeroBanner/HeroBanner.js

```diff
@@ -8,24 +7,27 @@ import {
   CaptionContainer,
   ButtonContainers,
   BlankContainer,
-} from './styles';
+} from "./styles";
+import Loader from "../loader";
```

Comment:
@sabithanazareth - I don't see this file - did you forget to check it in?

Reply:
This file already exists in the codebase. Here is the link: https://github.com/opportunity-hack/frontend-ohack.dev/blob/develop/src/components/loader.js

Reply:
good catch! This file shouldn't really exist anymore - will remove and change to a different loader 

here's what we normally use
```
return (<p>Loading... <Puff stroke="#0000FF" /> <Puff stroke="#0000FF" /></p>);
```

Reply:
Should I make these changes and commit them?

Reply:
No need! I'll update as I'm working through the merge conflict @sabithanazareth